Library/Formula/squid-deb-proxy.rb

```diff
@@ -0,0 +1,174 @@
+      # fix paths with the HOMEBREW_PREFIX
+      s.gsub! %r{([\s='"])/etc/}, "\\1#{etc}/"
+      s.gsub! %r{([\s='"])/var/}, "\\1#{var}/"
+    end
+
+    inreplace "init-common.sh" do |s|
+      # install should be as the calling user
+      s.gsub! /(\binstall\b.*?)(\s+-[og]\s*\S+)+/, "\\1"
+
+      # fix paths with the HOMEBREW_PREFIX
+      s.gsub! "/usr/sbin/squid", "#{HOMEBREW_PREFIX}/opt/squid/sbin/squid"
+      s.gsub! %r{([\s='"])/etc/}, "\\1#{etc}/"
+      s.gsub! %r{([\s='"])/var/}, "\\1#{var}/"
+    end
+
+    # pre-installation from debian/rules
+    # we take both sets because we're not either distro
+    IO.write(
+      "mirror-dstdomain.acl",
+      IO.read("mirror-dstdomain.acl.Debian") +
+        IO.read("mirror-dstdomain.acl.Ubuntu")
+    )
+
+    system "make", "install", "DESTDIR=#{prefix}"
+
+    # additional installation from debian/squid-deb-proxy.install
+    mkdir_p "#{prefix}/usr/share/squid-deb-proxy"
+    cp "init-common.sh", "#{prefix}/usr/share/squid-deb-proxy/"
+
+    # produce execution wrapper script
+    Pathname.new("#{sbin}/squid-deb-proxy").write(
+      <<-EOS.undent
+      #!/bin/bash
+
+      . '#{opt_prefix}/usr/share/squid-deb-proxy/init-common.sh'
+      pre_start
+
+      exec "$SQUID" -f '#{etc}/squid-deb-proxy/squid-deb-proxy.conf' "$@"
+      EOS
+    )
+
+    # produce placeholder files from debian/squid-deb-proxy.postinst
+    # these also serve to prevent the directories from being removed
+    user_conf      = "#{etc}/squid-deb-proxy/squid.conf.d/10-default.conf"
+    user_networks  = "#{etc}/squid-deb-proxy/allowed-networks-src.acl.d/10-default"
+    user_dests     = "#{etc}/squid-deb-proxy/mirror-dstdomain.acl.d/10-default"
+    user_blacklist = "#{etc}/squid-deb-proxy/pkg-blacklist.d/10-default"
+    Pathname.new(user_conf).write(
+      <<-EOS.undent
+      # #{user_conf}
+      #
+      # additional Squid configuration
+      EOS
+    )
+    Pathname.new(user_networks).write(
+      <<-EOS.undent
+      # #{user_networks}
+      #
+      # additional network sources that you want to allow access to the cache
+
+      # example net
+      #136.199.8.0/24
+      EOS
+    )
+    Pathname.new(user_dests).write(
+      <<-EOS.undent
+      # #{user_dests}
+      #
+      # network destinations that are allowed by this cache
+
+      # launchpad personal package archives (disabled by default)
+      #ppa.launchpad.net
+      #private-ppa.launchpad.net
+
+      # add additional mirror domains here (disabled by default)
+      #linux.dropbox.com
+      #download.virtualbox.org
+      #archive.getdeb.net
+      #packages.medibuntu.org
+      #dl.google.com
+      #repo.steampowered.com
+      EOS
+    )
+    Pathname.new(user_blacklist).write(
+      <<-EOS.undent
+      # #{user_blacklist}
+      #
+      # packages that should be not allowed for download, one binary packagename
+      # per line
+      #
+      #skype
+      EOS
+    )
+  end
+
+  test do
```

Comment:
This should be the very last part of the formula (Before the final end and start of the `__END__` patch)